guides/source/action_mailbox_basics.md

```diff
@@ -374,9 +420,19 @@ class ForwardsMailboxTest < ActionMailbox::TestCase
     recording = people(:david).buckets.first.recordings.last
     assert_equal people(:david), recording.creator
     assert_equal "Status update?", recording.forward.subject
-    assert_match "What's the status?", recording.forward.content.to_s
+    assert_match "What's the status?", recording.forward.decoded.to_s
```

Comment:
I believe this should still be `content`, as it's related to the `forward` Active Record model, `content` column. (Not the mail object) That's how the example above stores it: `content: mail.body.decoded` (or `content: mail.decoded` if we use the shortcut method mentioned as well)

```suggestion
    assert_match "What's the status?", recording.forward.content.to_s
```